Also includes emails, changes name

madssj opened the pull request:
I've pulled boosh's patch for emails and changed the project name to reflect that fact.

I'm not sure the correct thing is for you to pull this project, but if you think so please let me know so I can transfer ownership of the pipy app to you (or make you an admin).

Commit:
Database migration now changes the length of the email field and adds a unique index on email

Commit:
Monkey patching now applies patch to email field

madssj committed:
Merged (and adapted) patch from boosh

madssj committed:
Renamed app to longerusernameandemail

madssj committed:
Updated name change in README.md

madssj committed:
PEP8.

madssj committed:
Added discription in setup.py

madssj committed:
Fixed new package name

madssj committed:
Bumped release

dalazx committed:
django.contrib.auth.forms.AuthenticationForm doesn't have an email field. KeyError was raised. Fixed.
MAX_EMAIL_LENGTH was added.

madssj committed:
Merge pull request #1 from dalazx/master

Fixed KeyError and added MAX_EMAIL_LENGTH

madssj committed:
Added CHANGELOG

madssj committed:
Bumped version to 0.5.2

mfogel committed:
Enforcing required email

mfogel committed:
Standardize username help text

mfogel committed:
Fixing django admin 'add user' page

Previously, it would throw a 500 error.

madssj committed:
Merge pull request #2 from mfogel/master

Enforce email required

madssj committed:
Release 0.5.3

mfogel committed:
Tests for admin functions

madssj committed:
Merge pull request #3 from mfogel/feature/fix-admin-pages

Merged fix for admin create user form and unittests.

madssj committed:
Added changelog for 0.5.4 and bumped version in setup

skoczen commented:
Hey there,

Looks great - given that it's taken me this long to get to your pull request, I clearly don't have enough time to keep this updated on an ongoing basis.

Since you've got the more updated app, any problem if I update the docs on this project to point folks to yours?

Cheers, 
Steven

mfogel committed:
Tests for UserCreationForm

mfogel committed:
Set unique for User.email field

madssj committed:
Merge pull request #4 from mfogel/feature/fix-usercreationform

Feature/fix usercreationform

madssj committed:
Update setup.py

Updated version number.

madssj committed:
Update CHANGELOG.md

skoczen commented:
Hey @madssj - doing a review of the projects I'm maintaining right now.  Based on what I'm seeing in [your repo](https://github.com/madssj/django-longer-username-and-email), you seem to have a handle on keeping the codebase up.

I think I'd like to update the readme here to deprecate `longerusername` in favor of your project, and point people there.  Is that ok by you?

madssj commented:
Feel free to do so. Thankfully it won't be needed post 1.5 :)
Den 02/12/2012 20.30 skrev "Steven Skoczen" notifications@github.com:

> Hey @madssj https://github.com/madssj - doing a review of the projects
> I'm maintaining right now. Based on what I'm seeing in your repohttps://github.com/madssj/django-longer-username-and-email,
> you seem to have a handle on keeping the codebase up.
> 
> I think I'd like to update the readme here to deprecate longerusername in
> favor of your project, and point people there. Is that ok by you?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/GoodCloud/django-longer-username/pull/13#issuecomment-10933802.

skoczen committed:
- Changed uniqueness of email to be a setting, `REQUIRE_UNIQUE_EMAIL`.  Default is still True.
- Updates Readme to list all settings.
- Version bump

denilsonsa committed:
Django 1.5 already supports custom User models.

Adding a note (and links) about custom User model support in Django 1.5.

Submitting this here as per skoczen suggestion:
https://github.com/GoodCloud/django-longer-username/pull/18

madssj committed:
Merge pull request #6 from denilsonsa/patch-1

Django 1.5 already supports custom User models.

ErwinJunge committed:
Fix https://github.com/skoczen/django-longer-username/issues/15

madssj committed:
Merge pull request #10 from ErwinJunge/master

Fix https://github.com/skoczen/django-longer-username/issues/15

madssj committed:
Bumped version.

frost-nzcr4 committed:
Fix links and add note about increased lengths

madssj committed:
Merge pull request #11 from frost-nzcr4/update-readme

Fix links and add note about increased lengths
